docs/guides/src/main/server/db.adoc

```diff
@@ -73,57 +72,66 @@ Database schema only accounts for Unicode strings in the following special field
 
-=== Oracle database
+If the database character set was not created with Unicode support, you need to configure the JDBC driver to support Unicode characters in the special fields. You configure two properties. Note that you can configure these properties as system properties or as connection properties.
 
-Unicode characters are properly handled provided the database was created with Unicode support in VARCHAR and CHAR fields (e.g. by using AL32UTF8 character set as the database character set). No special settings is needed for JDBC driver.
+. Set `oracle.jdbc.defaultNChar` to `true`.
++
+[NOTE]
+====
+Setting this property might negatively impact performance. For more information, see the Oracle JDBC driver configuration documentation.
+====
 
-If the database character set is not Unicode, then to use Unicode characters in the special fields, the JDBC driver needs to be configured with the connection property oracle.jdbc.defaultNChar set to true. It might be wise, though not strictly necessary, to also set the oracle.jdbc.convertNcharLiterals connection property to true. These properties can be set either as system properties or as connection properties. Please note that setting oracle.jdbc.defaultNChar may have negative impact on performance. For details, please refer to Oracle JDBC driver configuration documentation.
+. Optionally, set `oracle.jdbc.convertNcharLiterals` to `true`.
 
-=== Microsoft SQL Server database
+=== Unicode support for a Microsoft SQL Server database
 
-Unicode characters are properly handled only for the special fields. No special settings of JDBC driver or database is necessary.
+Unicode characters are supported only for the special fields for a Microsoft SQL Server database. The JDBC driver and database require no special settings.
 
-=== MySQL database
+=== Configuring Unicode support for a MySQL database
 
-Unicode characters are properly handled provided the database was created with Unicode support in VARCHAR and CHAR fields in the CREATE DATABASE command (e.g. by using utf8 character set as the default database character set in MySQL 5.5. Please note that utf8mb4 character set does not work due to different storage requirements to utf8 character set [1]). Note that in this case, length restriction to non-special fields does not apply because columns are created to accommodate given amount of characters, not bytes. If the database default character set does not allow storing Unicode, only the special fields allow storing Unicode values.
+Unicode characters are supported in a MySQL database if the database was created with Unicode support in the VARCHAR and CHAR fields when using the CREATE DATABASE command. In other words, `utf8` was selected as the default database character set in MySQL 5.5.
```

Comment:
This rewording is misleading. `utf8` is just one of the options for Unicode support in MSSQL, and while still valid, it is deprecated now. Also the version 5.5 is an example of MySQL version, not the one that is in any way suggested to be used, and now is rather old. That was the reason for the "e.g." in the original text.